perimysium/dataman.py

```diff
@@ -409,6 +409,18 @@ def write(self, fpath):
 
         f.close()
 
+    def addnoise(self, noiseWidth):
```

Comment:
Oh and I don't think you want to return `self`, do you? You can just continue using the object.